docs/examples/example_satellite_observations.ipynb

```diff
@@ -0,0 +1,360 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    dy0 = 1852*60*np.abs(ds['latitude'][2].data - ds['latitude'][1].data)
</pre><p>Here it could be helpful to comment that the <code>dy</code> spacing is constant over the globe.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocean-eddy-cpt/gcm-filters/pull/92/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>